edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -39,7 +39,7 @@
     </p>
     <% } %>
     <% if (error) { %>
-      <p class="error"><%= error %></p>
+      <p class="error" tabindex="1"><%= error %></p>
```

Comment:
Should this be `tabindex="-1"` -- based on Mark Sadecki's comments on your other accessibility work?